core/src/main/java/io/ultrabrew/metrics/util/DistributionBucket.java

```diff
@@ -0,0 +1,66 @@
+package io.ultrabrew.metrics.util;
+
+public class DistributionBucket {
+
+    private static final String UNDERFLOW = "underflow";
+    private static final String OVERFLOW = "overflow";
+
+    private int[] buckets;
+
+    public DistributionBucket(final int[] buckets) {
```

Comment:
need to throw error if only 1 bucket is passed

Reply:
also validate if buckets aren't in increasing order, or if 2 entries have same value

Reply:
or sort the values and eliminate duplicates